src/main/java/org/folio/rest/client/OkapiClient.java

```diff
@@ -30,6 +30,8 @@
 import io.vertx.ext.web.client.WebClient;
 
 public class OkapiClient {
+  public static final String EMPTY_STRING = "";
```

Comment:
StringUtil.EMPTY_STRING

Reply:
Ok, thanks, it is done

Reply:
I have used org.apache.commons.lang3.StringUtils